### ISSUE TITLE
Indicate payment

### ISSUE THREAD

**Issue by okainov** [giggls/opencampsitemap#18]
Background:
I almost never ever stay in any place which asks for payment. I would love to use this website to find camping/overnight place, but I really miss some kind of indication or filtering by payment.

I think in OSM there is already a tag like `pay` or so, so it should be possible to show it.

**Comment by giggls** [comment]
Yes, if tagged the key fee is even in the already transmitted json data.

Frankly I decided not to show it for two reasons:

1. If fee=yes is not tagged there is likely a fee to be payed in most cases anyway.
I suppose fee=no is the exception rather than the rule.
2. I do not even have an idea for a fancy icon :-)

It would make certainly make more sense to show places tagged fee=no rather than the ones tagged fee=yes. This said if someone will draw an icon for me I will add it anyway.

Or should we just add an additional sentence like this:

**A fee is charged for using this site.**
**No fee is charged for using this site.**

Would need to have this in 5 languages though.

**Comment by okainov** [comment]
>If fee=yes is not tagged there is likely a fee to be payed in most cases anyway.
I suppose fee=no is the exception rather than the rule.

Well, it depends on how you see it. If you look at "proper" campsite - sure they usually would have payment.

But I look more at wild camping, dispersed camping, informal/backcountry "campsites" and there it's the other way around. So for me it's opposite :)

Just adding text would be at least something. And if there would be possibility to filter, I'm not even sure extra icons are needed

**Comment by okainov** [comment]
>Frankly I doubt that my map is even the right tool for what you are looking for.

Ok, maybe I misunderstood the purpose of this map then... I saw this as a tool which can be used by travelers to find camping places. And as well as by mappers-travelers who are at some campsite now and they don't see it existing in OSM yet... But if your idea was not to have it explorable, then it's probably understandable why filters are not there...

**Comment by giggls** [comment]
The purpose of this map is to have a map **at all** in the first place to make the campsite data in OSM more usable than it was back in 2019 and unfortunately still is in many places.
See my announcement from 2019:
https://blog.gegg.us/2019/01/announcing-open-camping-map/

The feature you are requesting here sounds like a niche topic to me and does not make that much sense IMO anyway given the 8.72% coverage.

Higher on my to do list is some kind of export feature for Apps like Osmand and a possibility to have a campsite-layer on [Brouter Web](http://brouter.de/brouter-web/).

Like always with FOSS. Feel free to send patches if you want to have a particular feature I am not interested in myself that much.

**Comment by giggls** [comment]
Add some time ago. Forgot to close.
